shared_model/interfaces/commands/create_account.hpp

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef IROHA_SHARED_MODEL_CREATE_ACCOUNT_HPP
+#define IROHA_SHARED_MODEL_CREATE_ACCOUNT_HPP
+
+#include "interfaces/common_objects/types.hpp"
+#include "interfaces/primitive.hpp"
+#include "model/commands/create_account.hpp"
+
+namespace shared_model {
+  namespace interface {
+
+    /**
+     * Create acccount in Iroha domain
+     */
+    class CreateAccount
+        : public Primitive<CreateAccount, iroha::model::CreateAccount> {
+     public:
+      /// Type returned by accountName method
+      using AccountNameType = std::string;
```

Comment:
same, should be in types.hpp

Reply:
plz, do for other types as well

Reply:
Account Name is used only in this command, the rest use AccountIdType